### PR TITLE
[5.2] Add custom css

### PR DESCRIPTION
**Pull request by rdeutz**
Pull Request for Issue #41205 .

### Summary of Changes

Checked if class is set and added it to the fields

### Testing Instructions

copied from the issue

```php
$options=array();
$o = new stdClass();
$o->value = 1;
$o->text = 'option 1';
$options[] = $o;

$attribs=array();
$attribs['class'] = 'MyCLass'
HTMLHelper::_('select.radiolist', $options, 'fieldname', $attribs);
```
### Actual result BEFORE applying this Pull Request

```php
<div class="controls">
  <div class="form-check form-check-inline">
    <input type="radio" class="form-check-input"  name="fieldname" id="fieldname1" value="1" class="MyCLass">
    <label for="fieldname1" class="form-check-label" id="fieldname1-lbl">option 1</label>
  </div>
</div>
```
class attribute is duplicated

### Expected result AFTER applying this Pull Request

```php
<div class="controls">
  <div class="form-check form-check-inline">
    <input type="radio" class="form-check-input MyCLass"  name="fieldname" id="fieldname1" value="1">
    <label for="fieldname1" class="form-check-label" id="fieldname1-lbl">option 1</label>
  </div>
</div>
```

Both the form-check-input and MyCLass should be merged in one class attribute

### Link to documentations
Please select:
- [ ] Documentation link for docs.joomla.org: <link>
- [ ] No documentation changes for docs.joomla.org needed

- [ ] Pull Request link for manual.joomla.org: <link>
- [ ] No documentation changes for manual.joomla.org needed

### CODE REVIEW

File: libraries/src/HTML/Helpers/Select.php [hunk 1]
```diff
@@ -631,7 +631,15 @@ public static function radiolist(
         $idtag = false,
         $translate = false
     ) {
+        $class = '';
+
         if (\is_array($attribs)) {
+
+            if (array_key_exists('class', $attribs)) {
```
Comment (richard67) [inline libraries/src/HTML/Helpers/Select.php]:
```suggestion
            if (\array_key_exists('class', $attribs)) {
```
Fix PHPCS (php-cs-fixer) in drone - native_function_invocation

File: libraries/src/HTML/Helpers/Select.php [hunk 2]
```diff
@@ -631,7 +631,15 @@ public static function radiolist(
         $idtag = false,
         $translate = false
     ) {
+        $class = '';
+
         if (\is_array($attribs)) {
+
```
Comment (richard67) [inline libraries/src/HTML/Helpers/Select.php]:
```suggestion
```
Now as the one PHP code style checker is happy, the oner one complains. Here it wants us to remove the empty line at the top inside a control structure.

File: libraries/src/HTML/Helpers/Select.php [hunk 3]
```diff
@@ -662,7 +670,7 @@ public static function radiolist(
                 $extra .= ((string) $k === (string) $selected ? ' checked="checked" ' : '');
             }
 
-            $html .= '<input type="radio" class="form-check-input" name="' . $name . '" id="' . $id . '" value="' . $k . '" '
+            $html .= '<input type="radio" class="form-check-input' . $class .'" name="' . $name . '" id="' . $id . '" value="' . $k . '" '
```
Comment (richard67) [inline libraries/src/HTML/Helpers/Select.php]:
```suggestion
            $html .= '<input type="radio" class="form-check-input' . $class . '" name="' . $name . '" id="' . $id . '" value="' . $k . '" '
```
PHPCS - this should be the last one.

**Comment by rdeutz** [comment]
Thanks @richard67

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on 89324881ba52daf937a042b7af0645353df2e0ef<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/43392">issues.joomla.org/tracker/joomla-cms/43392</a>.</sub>

**Comment by alikon** [comment]
I have tested this item :white_check_mark: successfully on 89324881ba52daf937a042b7af0645353df2e0ef<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/43392">issues.joomla.org/tracker/joomla-cms/43392</a>.</sub>

**Comment by alikon** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/43392">issues.joomla.org/tracker/joomla-cms/43392</a>.</sub>

**Comment by rdeutz** [comment]
just for the record,  I used this PR to pin the mysql service in our testing to 8.3.0 because it seems we have a problem with 8.4 and the integration tests

**Pull request merged**